assets/panel/debugger.properties

```diff
@@ -24,6 +24,11 @@ copySource.accesskey=y
 copySourceUri2=Copy source URI
 copySourceUri2.accesskey=u
 
+# LOCALIZATION NOTE (setDirectoryRoot): This is the text that appears in the
```

Comment:
setDirectoryRoot -> setDirectoryRoot.label

Reply:
@jasonLaster this was not fixed before merging!

Reply:
oops - sorry i missed this :/